docs/DeveloperGuide.md

```diff
@@ -154,90 +155,44 @@ Classes used by multiple components are in the `seedu.address.commons` package.
 
 This section describes some noteworthy details on how certain features are implemented.
 
-### \[Proposed\] Undo/redo feature
+Usage of `editClient` and `editMeeting`:
 
-#### Proposed Implementation
+`editClient` and `editMeeting` execute in a similar manner to each other.
+Let the term entity refer to either a client or a meeting.
+Every entity is uniquely identified by a UUID in storage, so essentially all fields in an entity can be edited without loss of uniqueness of the entity.
 
-The proposed undo/redo mechanism is facilitated by `VersionedMyInsuRec`. It extends `MyInsuRec` with an undo/redo history, stored internally as an `myInsuRecStateList` and `currentStatePointer`. Additionally, it implements the following operations:
+[Proposed] Multiple possible prefixes per command feature
```

Comment:
Just want to clarify, is this a feature that u are hinting to the developer reading this that it is one feature that they can implement?

Reply:
yes, a proposed feature that can be added like undo/redo mentioned in AB3